cypress/integration/modal.spec.js

```diff
@@ -22,4 +23,11 @@ describe('when the server responds successfully', () => {
     cy.findByTestId(CLOSE_BUTTON).click();
     cy.findByTestId(MODAL_CONTENT).should('not.exist');
   });
+
+  it('closes the modal when the overlay is clicked', () => {
```

Comment:
Let's also add a test for clicking the close button.

Reply:
After adding that test, I think it'd be best to change the other tests to use the close button instead of the overlay. It's good to have regression coverage for clicking the overlay in the modal spec specifically, but force clicks in Cypress can be flakey. I think it'd be better to target the actual close button for other integration tests.

Reply:
Oh when I looked there already was a test for the close button written right before this overlay one! And that sounds good, I'll switch those out in the other page tests.